src/equations/svaerd_kalisch_1d.jl

```diff
@@ -189,7 +188,8 @@ function create_cache(mesh, equations::SvaerdKalischEquations1D,
     else
         @error "unknown type of first-derivative operator: $(typeof(solver.D1))"
     end
-    return (hmD1betaD1 = hmD1betaD1, D1betaD1 = D1betaD1, D = D, h = h, hv = hv,
+    factorization = cholesky(Diagonal(ones(nnodes(mesh))) - D1betaD1)
```

Comment:
Is there a more elegant way of pre-allocating the `factorization`?

Reply:
I don't think so 🤷

Reply:
Ok, then let's keep it like this.